demos/filebench/dl_and_build_filebench.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+echo -e "${BLUE}Start installing dependencies.${NC}"
+
+DEPS="bison flex"
+
+apt-get update
+apt-get install -y $DEPS
+echo -e "${BLUE}Finish installing dependencies.${NC}"
+
+echo -e "${BLUE}Start building filebench from tarball.${NC}"
+
+# Download release tarball
+VERSION="1.5-alpha3"
+TARBALL="filebench-${VERSION}.tar.gz"
+rm -f ${TARBALL}
+wget https://github.com/filebench/filebench/releases/download/${VERSION}/${TARBALL}
+rm -rf filebench && mkdir filebench
+tar -zxvf filebench-${VERSION}.tar.gz -C filebench --strip-components 1
+pushd filebench
+
+# Make modification to
+# 1. Replace fork to vfork
+# 2. Prepare shared memory region for child processes
+# 3. Disable SYSV semaphores
+sed -i '93d' procflow.c
+sed -i '93 i  if ((pid = vfork()) < 0) {' procflow.c
+sed -i '418 i  filebench_shm = (filebench_shm_t *)shmaddr;' ipc.c
+sed -i '403,417d' ipc.c
+
+./configure
+sed -i '298 i  #undef HAVE_SYSV_SEM' config.h
```

Comment:
Using sed to modify the source code does not seem to be maintainable. How about having diff file? So that in this script we can simply apply the patch.

Reply:
Make sense to me. I had thought that "diff+patch" is a git feature.